src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +12,8 @@
  */
 public class Constants {
     //Put constants in this class using "public static double CONSTANT_NAME"
+    public enum driveType{
```

Comment:
This is not being used and can be removed.

Reply:
done